content/en/tracing/troubleshooting/tracer_startup_logs.md

```diff
@@ -165,6 +165,9 @@ The Go Tracer prints one of two possible diagnostic lines, one for when the Agen
 {{< /programming-lang >}}
 {{< programming-lang lang="nodejs" >}}
 
+Startup logs are disabled by default starting in verson 2.X+ of the tracer. They can enabled using the environment variable `DD_TRACE_STARTUP_LOGS=true`.
```

Comment:
Was there any mention of the startup logs previously? It seems to me like this sentence is not super useful with its current location.

Reply:
I added this sentence so as not to cause confusion on why Startup logs aren't present as they used to be enabled by default. We have this [paragraph at the bottom](https://docs.datadoghq.com/tracing/troubleshooting/tracer_startup_logs/#disabling-startup-logs) that on how to disable startup logs but this section is present for all languages so would need a bigger document overhaul to change this.